ember-slugify/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "ember-slugify",
```

Comment:
When trying to test this branch on another repo I got following error
```
error Can't add "ember-slugify": invalid package version undefined.
```

Do you managed to test it on another repo?